app/models/event.rb

```diff
@@ -0,0 +1,15 @@
+class Event < ApplicationRecord
+  validates :title, :place_title, :place_cord, :description, :start_at, :end_at, presence: true
+  validates :title, length: { in: 6..20 }
+  validates :place_title, length: { maximum: 30 }
+  validates :description, length: { minimum: 30 }
```

Comment:
Why is length the same for place description and title? Anyway, if so, you can combine these two validations into one

Reply:
I see, sorry, my fault.